deployTest/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.ibissource</groupId>
+		<artifactId>ibis-adapterframework-parent</artifactId>
+		<version>7.6-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>ibis-adapterframework-assemble-test</artifactId>
+	<name>Ibis AdapterFramework assemble test</name>
+
+	<properties>
+		<mavenDependencyPluginVersion>3.1.2</mavenDependencyPluginVersion>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.ibissource</groupId>
+			<artifactId>ibis-adapterframework-test</artifactId>
+			<version>7.6-SNAPSHOT</version>
+			<type>war</type>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.geronimo.specs</groupId>
+			<artifactId>geronimo-jms_1.1_spec</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-dbcp</groupId>
+			<artifactId>commons-dbcp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-pool</groupId>
+			<artifactId>commons-pool</artifactId>
+		</dependency>
```

Comment:
uit eindelijk gaat het puur hier om, de dependencies voor de betreffende databases. 
Dit zou ook in de iaf test module kunnen. Er zit niet heel veel overhead/winst in het lostrekken van de profielen naar een andere module.

Reply:
Opgevolgd.